lib/puppet/version.rb

```diff
@@ -5,6 +5,11 @@
 # The version can be set programmatically because we want to allow the
 # Raketasks and such to set the version based on the output of `git describe`
 
+# Seems to be the minimum to get a vendored library without adding extra dependencies.
+vendor_dir = File.join([File.dirname(File.expand_path(__FILE__)), "vendor"])
+load "#{vendor_dir}/load_semantic_puppet.rb"
```

Comment:
Yeah that was my concern... how bad would it feel to reach into vendor directly. Seems iffy at best, risky at worst. I think +1 to just split/join, but can you make sure the variable names match up with SemanticPuppet?